kwokfixture/kwokfixture.go

```diff
@@ -0,0 +1,55 @@
+package kwokfixture
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/redhat-appstudio/segment-bridge.git/testfixture"
+)
+
+const (
+	contextName      = "kwok"
+	clusterServerURL = "http://127.0.0.1:8080"
+	scriptPath       = "../scripts/get-uid-map.sh"
```

Comment:
The `kwokfixture` package shouldn't hard code this script or else it can't be reused easily. It should be passed in by the test.